lib/kamal/configuration/env.rb

```diff
@@ -24,7 +28,13 @@ def secrets_io
   end
 
   def secrets
-    @secrets ||= secrets_keys.to_h { |key| [ key, ENV.fetch(key) ] }
+    # TODO: More than one @env_file
+    # TODO: Considerer a merge between env_file and env
+    if @env_file
+      Dotenv::Environment.new(@env_file)
+    else
+      secrets_keys.to_h { |key| [ key, ENV.fetch(key) ] }
+    end
```

Comment:
This is the main goal. Not sure how we can tackle this.
The reason behind this is to push an env file with "gazillions" of environment variables that can be related to the application code or not.

Any thoughts @djmb ? (sorry to tag you, but I saw that you did the last merges around here)